### PR TITLE
chore: Corrects Read Tab Content

### PR DESCRIPTION
**Pull request by redreceipt**
Shows the latest devotional as the featured item at the top of the Read tab

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -235,10 +235,11 @@ HOME_FEATURES:
...
-  - algorithms:
+  - subtitle: Read
+    algorithms:
       - type: CONTENT_CHANNEL
         arguments:
-          contentChannelId: 18
+          contentChannelId: 17
           limit: 1
     type: VerticalCardList
   - title: Continue Reading
```
Comment (vinnyjth) [inline config.yml:245]:
```suggestion
  - subtitle: Continue Reading
```

Comment (vinnyjth) [reply config.yml:245]: I can't comment on line 251 - but that should probably also be `title`

Comment (redreceipt) [reply config.yml:245]: the mock shows different sizes is why I did it.

Comment (vinnyjth) [reply config.yml:245]: ah - in this case I think the all the same size looks better than using the `title`s for the two lower features.

Comment (redreceipt) [reply config.yml]: done

**Review by vinnyjth — APPROVED** [review]
💥

**Pull request merged**